### DISCUSSION TITLE
Clips clean up options?

### DISCUSSION THREAD

**Discussion by tomlut** [tphakala/birdnet-go#143] [General]
Just started a brand new instance but copied across my old config file. I am seeing this logged a lot:
```
Error retrieving clips for removal: invalid parameters: minHours and minClips must be greater than 0
```

Where do these options go in the config file?

I don't see them in the wiki example.

**Comment by tomlut** [comment]
Found it. https://github.com/tphakala/birdnet-go/blob/8ab36fb4d8e21fb60cc8c5ca799f6d765f9c78b5/internal/conf/config.go#L288C1-L292C99

**Comment by tomlut** [comment]
Actually that did not help. Still getting:
```
Error retrieving clips for removal: invalid parameters: minHours and minClips must be greater than 0

```

**Comment by tphakala** [comment]
Audio clip retention has been completely reworked in v0.5.4 release, anyone stumbling on this discussion please see v0.5.4 release discussion for more details.
